bigquery_etl/schema/__init__.py

```diff
@@ -127,6 +127,13 @@ def compatible(self, other: "Schema") -> bool:
 
         return True
 
+    @staticmethod
+    def _node_with_mode(node):
+        """Add default value for mode to node."""
+        if "mode" in node:
+            return node
+        return {"mode": "NULLABLE", **node}
```

Comment:
mode=NULLABLE is missing in the schema returned from dry running a `CREATE TABLE` statement